docker_cli/commands.py

```diff
@@ -0,0 +1,119 @@
+import os
+from shlex import split
+from subprocess import DEVNULL, PIPE, run
+from typing import Iterable, List, Union
+
+from ._docker_mamba import mamba_lockfile_command
+from ._image import Image
+from ._utils import is_conda_pkg_name, universal_tag_prefix
+
+
+def dropin(tag: str) -> None:
+    """
+    Initiates a drop-in session on an image.
+
+    Parameters
+    ----------
+    tag : str
+        The tag or ID of the image.
+    """
+    image: Image = Image(tag)
+
+    image.drop_in()
+
+
+def remove(
+    tags: Iterable[str],
+    force: bool = True,
```

Comment:
`--force` should not be the default IMO. I think it's more intuitive to use the default behavior of `docker rmi` as our own defaults

```suggestion
    force: bool = False,
```